steamclog/src/main/java/com/steamclock/steamclog/Steamclog.kt

```diff
@@ -55,9 +55,10 @@ object SteamcLog {
     }
 
     fun initWith(isDebug: Boolean,
-                 fileWritePath: File? = null) {
+                 fileWritePath: File? = null,
+                 fileRotationSeconds: Long? = null) {
         fileWritePath?.let {
-            this.config = Config(isDebug, fileWritePath)
+            this.config = Config(isDebug, fileWritePath, autoRotateConfig = AutoRotateConfig(fileRotationSeconds ?: AutoRotateConfig.defaultFileRotationSeconds))
```

Comment:
If AutoRotateConfig already contains a default for fileRotationSeconds, do we need to set it here?

Reply:
I think we still need to set it in both places, but I've cleaned it up a bit!